### DISCUSSION TITLE
Is this possible with useFieldArray?

### DISCUSSION THREAD

**Discussion by nani1008** [react-hook-form/react-hook-form#5425] [Ideas]
I want to implement this using react hook form with useFieldArray, is this possible? Any guides or tips on how to produce this? Thank you! 

https://www.jqueryscript.net/demo/multiple-email-input/

**Comment by bluebill1049** [comment]
I would build that with a controlled input and wrapped with Controller.

**Reply by nani1008** [reply]
Any base guides to do this?

**Reply by thanh-nguyen-95** [reply]
You can follow this article as a base. It's basically a tag input.

https://blog.logrocket.com/building-a-tag-input-field-component-for-react/
